Gruntfile.js

```diff
@@ -27,7 +27,7 @@ module.exports = function (grunt) {
         },
         browserify: {
             client: {
-                src: ['src/Frontend/**/*.js', '!src/Frontend/__tests__/*.js'],
+                src: ['src/Frontend/**/*.js', '!src/Frontend/__tests__/**/*.js '],
```

Comment:
you have to add the PathSpec *src/Frontend/App.js* at the end of the src list in order to tell browserify to render the App.js